docs/index.html

```diff
@@ -1,14 +1,18 @@
 <!DOCTYPE HTML>
 <html lang="en-US">
-    <head>
-        <meta charset="UTF-8">
-        <meta http-equiv="refresh" content="0; url=https://agentos.org/latest/">
-        <script type="text/javascript">
-            window.location.href = "https://agentos.org/latest/"
-        </script>
-        <title>Page Redirection</title>
-    </head>
-    <body>
-        If you are not redirected automatically, click to visit the <a href='https://agentos.org/latest/'>most recent agentos docs</a>.
-    </body>
-</html>
+
+<head>
```

Comment:
Any reason why you decided to remove the indentation of the head and body text so that they are now at the same layer as their container tag (the <html> tag)?

Reply:
I think it was my default formatter. I will add the indentation and commit the changes.